lib/main.dart

```diff
@@ -427,14 +426,44 @@ void addToAmplitude() async {
   var lambdaEndpt =
       'https://jsr2rwrw5m.execute-api.us-east-1.amazonaws.com/default/pb-lambda-microservice';
 
-  var body = json.encode({
+  var eventProperties = MainInfo().amplitudMap['eventProperties'];
+
+  eventProperties['eggs'] = PBPluginListHelper.names ?? [];
+
+  eventProperties['Design specification'] =
```

Comment:
Ideally all of this would be done in the analytics service. We should see if one exists already and find a way to encapsulate this logic to avoid giving more responsibilities to `main.dart`